db/schema.rb

```diff
@@ -34,6 +32,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "goals", force: :cascade do |t|
```

Comment:
This table does not belong in this PR. Try and think of it as keeping each commit as a fully working application.
In this case, there will be a `goals` table without any other application support for its existence.

Reply:
I don't know why this happens, I have a goals model branch which has this migration and table in the schema. For some reason if tables and migrations exist in other branches they seem to appear in a new branch. I didn't create the goals migration in this branch???